docs/conf.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
@seanlip, should we also add the copyright notice here?

Reply:
No, I don't think it's needed here, since this is a config file that's mostly not written by us.